modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/ReferringURL.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.analytics.reports.web.internal.model;
+
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+
+import java.util.Objects;
+
+/**
+ * @author David Arques
+ */
+public class ReferringURL {
+
+	public ReferringURL() {
+	}
+
+	public ReferringURL(int traffic, String url) {
```

Comment:
I think we are calling the traffic, trafficAmount, in other  classes

Reply:
true! It's the frontend representation which has this name!

Reply:
Probably we should change it in the frontend too.

/cc @dgarciasarai

Reply:
In the frontend for each countryKeywords I have: keyword, position, searchVolume and traffic. 
For the PieChart we have traffic volume and traffic share but it comes from `trafficSource.value` and `trafficSource.share`. What is the problem here? So I can understand it better.

Reply:
The problem is that sometimes we use `traffic` and sometimes `trafficAmount`, we should choose one word, and use always the same if we talk about the same concept, otherwise is very confusing.

@darquesdev says that the reason we are doing this in the back is because of a fronted contract we have, so if this is the reason, we should change the frontend contract

Reply:
@dgarciasarai finally I'm using `trafficAmount`!

Reply:
hey @cgoncas, should we change the existing "traffic" property used in `SearchKeyword` see https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/model/SearchKeyword.java#L99 both in the back and front end?